tests/data/ATL16_prefetch.dmr

```diff
@@ -222,4 +222,4 @@
     <Attribute name="short_name" type="String">
         <Value>ATL16</Value>
     </Attribute>
-</Dataset>
\ No newline at end of file
+</Dataset>
```

Comment:
...and this one was from the blank line check. Good evidence that one is working, too!